src/components/PopupWithForm.js

```diff
@@ -5,6 +5,9 @@ export default class PopupWithForm extends Popup {
     this._popupForm = document.querySelector(popupModalSelector);
```

Comment:
`    this._popupForm = document.querySelector(popupModalSelector);
`

Please, delete it

**No** need to find the popup again in the inheritors of class `Popup` because the parent has already found it in its constructor and it’s `this._popupModalSelector`. It’s available in any inheritor.